From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Thu, 11 Jun 2015 18:26:16 -0400
Subject: avoid identifying as beta

If we build from git-buildpackage using autoreconf, it comes out
looking like a beta because we're not on a git tag.

This patch avoids that situation.
---
 autogen.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff -Naur gpgme-1.19.0/autogen.sh gpgme-1.19.0.tpg/autogen.sh
--- gpgme-1.19.0/autogen.sh	2023-03-15 15:05:10.000000000 +0000
+++ gpgme-1.19.0.tpg/autogen.sh	2023-04-13 19:24:01.789145403 +0000
@@ -243,7 +243,7 @@
     fi
 
     beta=no
-    if [ -e .git ]; then
+    if false; then
       ingit=yes
       tmp=$(git describe --match "${matchstr1}" --long 2>/dev/null)
       if [ -n "$tmp" ]; then
@@ -268,8 +268,8 @@
       rvd=$((0x$(echo ${rev} | dd bs=1 count=4 2>/dev/null)))
     else
       ingit=no
-      beta=yes
-      tmp="-unknown"
+      beta=no
+      tmp=""
       rev="0000000"
       rvd="0"
     fi
